pages/EC-WebLogic_help.xml

```diff
@@ -200,6 +200,10 @@
                         <td>JAVA_HOME environment variable. It should be used when java is not in the path, or different java need to be used.</td>
                     </tr>
                     <tr>
+                      <td>JAVA_VENDOR</td>
+                      <td>JAVA_VENDOR environment variable. It should be used when java vendor need to be specified.</td>
```

Comment:
JAVA_VENDOR environment variable. It should be used to specify the Java vendor value when setting the JAVA_HOME parameter to point to a specific JDK.